### ISSUE TITLE
Some strings are unused

### ISSUE THREAD

**Issue by didib** [AndBible/and-bible#2422]
TL;DR: Might it be possible to write and use an automated tool that will find unused strings, and perhaps remove them?

I understand that removal is somewhat delicate, and will require human inspection. E.g. if a patch removes the use of a string, but there are plans to re-introduce the use soon thereafter, it does not make sense to remove it, and especially not its translations.

See e.g. jumpToStudyPad, studyPadModalTitle.

For jumpToStudyPad, I found the original code adding and then removing it, by checking the git log on app/bibleview-js/src/lang/default.yaml . It was added on ab0beaa7, and this patch also added the code using it, to app/bibleview-js/src/components/LabelList.vue . git log on latter found where it was removed, 9a8c3673. Based on ab0beaa7, I now decided to translate it to Hebrew as something like, if back-translated, "transfer to _the_ Study Pad" (IMO in Hebrew it will sound slightly less natural to omit the "the". I also decided that "jump", in Hebrew, will sound too slang-y, to me, at least).

With studyPadModalTitle, I didn't manage to use the same approach, because IIUC the code using it was added in more than one patch (0918be85b is only JS, does not use it directly), and so I have to spend more time trying to understand it all works (or somehow git log on the entire project, and skip over translations...).

**Comment by tuomas2** [comment]
conversion to typescript together with ide will help to identify unused strings (#2411)

**Comment by tuomas2** [comment]
Fixed in 1727ba30b81bc4c41645c58b44cf6cee24c770ea (won't be fixed in Transifed until we enable translations there towards 4.1).
